lib/src/main/java/dev/pcvolkmer/onko/obds2to3/StrahlentherapieMapper.java

```diff
@@ -206,22 +212,21 @@ private static Applikationsart mapApplikationsart(Bestrahlung bestrahlung) {
 
             kontakt.setEinzeldosis(einzeldosis);
             kontakt.setGesamtdosis(gesamtdosis);
+            kontakt.setInterstitiellEndokavitaer("K");
             switch (applikationsart) {
                 case "K":
```

Comment:
in obds v3 müsste eigentlich RateType (https://github.com/pcvolkmer/obds2-to-obds3/blob/master/lib/src/main/resources/schema/oBDS_v3.0.4.xsd#L4820) gesetzt werden, allerdings können wir das nur auf Basis von "K" mMn nicht befüllen. Damit wird das erzeugte obdsv3 nicht mehr schema-konform.